src/main/java/frc/robot/commands/autoCommands/AutoPath.java

```diff
@@ -20,7 +20,7 @@ public class AutoPath extends SequentialCommandGroup {
 
     // Creates path with default velocity settings.
     public AutoPath(String path) {
-        this(path, new PathConstraints(4, 3));
+        this(path, new PathConstraints(Constants.AutoConstants.maxVelocity));
```

Comment:
here you would reference 
Constants.AutoConstants.velocity, Constants.AutoConstants.acceleration
instead